packages/drivers/routerlicious-driver/src/errorUtils.ts

```diff
@@ -14,6 +14,7 @@ import { pkgVersion as driverVersion } from "./packageVersion";
 
 export enum R11sErrorType {
     fileNotFoundOrAccessDeniedError = "fileNotFoundOrAccessDeniedError",
+    sslCertError = "sslCertError",
```

Comment:
Just noting that this enum is not exported outside the package, so adding an entry to it is not a breaking change 👍 .